code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -520,6 +529,56 @@ the implant may become unstable and either pre-maturely inject the subject or si
 /obj/item/weapon/implant/compressed/islegal()
 	return 0
 
+
+//Cortical Implant
 /obj/item/weapon/implant/cortical
 	name = "cortical stack"
 	desc = "A fist-sized mass of biocircuits and chips."
+
+
+//Pax(Peace) Implant
+/obj/item/weapon/implant/peace
+	name = "pax implant"
+	desc = "A bean-shaped implant with a single embossed word - PAX - on it."
+	var/imp_alive = 0
+	var/msg_debounce = 0
+	var/imp_data = {"
+<b>Implant Specifications:</b><BR>
```

Comment:
Why Do People Type Like This? I Am Puzzled

Reply:
Because it's the title of the device's display page (I think?)